pkg/cmd/tracee.go

```diff
@@ -101,7 +101,7 @@ func PrintEventList(printRulesSet bool) {
 	printEventGroup(&b, events.InitNamespaces, events.MaxUserSpace)
 	b.WriteString("\n\nNetwork Events: " + titleHeaderPadFirst + "Sets:" + titleHeaderPadSecond + "Arguments:\n")
 	b.WriteString("____________  " + titleHeaderPadFirst + "____ " + titleHeaderPadSecond + "_________\n\n")
```

Comment:
BTW I'll fix these header pads as they are broken. There is only a single headerpad for multiple headers and the pads don't align. I'll create a very simple table with the appropriate module.

Reply:
https://github.com/aquasecurity/tracee/issues/2590